### ISSUE TITLE
How to get RF out of All in one Generator?

### ISSUE THREAD

**Issue by drugga** [IStrikeboomI/CobblestoneSemantics#8]
Hi, I have a All in one Generator running that produces ALOT of RF/FE buuut I have no idea how to use it. No matter what extraction block/cable/conduit I use from other mods, I am not able to use the generated RF anywhere. It is just stored inside the generator with no use for it? How can I use the energy created? Because right now it only functions as an expensive decorative block.

**Comment by IStrikeboomI** [comment]
What version of minecraft and the mod is this?

**Comment by drugga** [comment]
Minecraft 1.12.2 and the mod is 1.12.2 too.
I found the mod on the curseforge launcher while adding some mods to my MCEternal Lite modpack. If the mod is not supposed to be on that version then I am sorry for bothering but after googling the mod it took me here.

**Comment by IStrikeboomI** [comment]
I'm able to get power out of it using cyclic energy cables, what energy cables are you using?

**Comment by drugga** [comment]
I basically used it with everything.
Ender IO, IC2, Buildcraft and other mods. 
but I will test the cyclic cables.

Update:
I actually already tried the energy cables from cyclic before but
I had to use the Bundeled cables to get the power to transfer.
Thanks for the help.
Kinda feel like a fool for finding out that the solution is quite simple.
Sorry for the inconvinience.

**Comment by drugga** [comment]
Soooooo I started playing modded MC again and I now have the same issue.
I am playing the 1.16.5 ,,All in one, modded One Block" from NaturaSpell and added the Cobblestone Semantics mod through courseforge again (mod version 1.16.5-1.0.jar). Like last time I am unable to get the energy out of the all in one generator. I actually tried everything now. Cyclic cables and transmitters (in my cyclic version there is no bundled cable idk why), Draconic evolution transmitters, mekanism cables, pipez and probably more. I need help ;-;.
